trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py

```diff
@@ -53,11 +53,9 @@ def compressibility_drag_wing(state,settings,geometry):
     
     wing = geometry
     if wing.tag == 'main_wing':
-        wing_lifts = conditions.aerodynamics.lift_breakdown.compressible_wings # currently the total aircraft lift
```

Comment:
The lift of each wing is given individually in SUAVE. This is a poor assumption to assume the main wing is the only source of lift.

You can see the lift computation here:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py#L174

Reply:
I agree with this but I think it's better to have a clear assumption than to hide it.

Reply:
Eliminating the assumption is still noted as an enhancement in #309.